bazel/init.bzl

```diff
@@ -9,7 +9,7 @@ load("//bazel/ui:deps.bzl", "install_ui_deps")
 
 def enkit_init_go():
     go_rules_dependencies()
-    go_register_toolchains(version = "1.16")
+    go_register_toolchains(version = "1.17.1")
```

Comment:
So I specifically made it 1.16 because that's the most supported version of app engine. It should probably remain there until app engine supports it.

Reply:
Do you have a source? The docs I found suggest not even 1.16 is supported: https://cloud.google.com/appengine/docs/flexible/go/runtime#go_versions

Reply:
Reverted the go toolchain update - it's not necessary; I was just following the rules_go docs for their release.